js/components/icon.js

```diff
@@ -4,7 +4,7 @@ var React = require('react'),
 
 var Icon = React.createClass({
   shouldComponentUpdate: function(nextProps, nextState) {
-    return  (nextProps.icon != this.props.icon);
+    return  (nextProps.icon != this.props.icon || nextProps.className !== this.props.className);
```

Comment:
can the first part be changed to '!==' ?

Reply:
If not can you leave a comment saying why.  We are trying to move all the old code always use === and !== if possible.

Reply:
\+

Reply:
\+